tests/std/tests/P0896R4_ranges_alg_shuffle/test.cpp

```diff
@@ -0,0 +1,55 @@
+// shuffleright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <random>
+#include <ranges>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+
+using namespace std;
+
+mt19937 gen{random_device{}()};

```

Comment:
Ditto "fixed/default seed or output on error so we can reproduce."

Reply:
Ditto global